tests/factories/lti_user.py

```diff
@@ -0,0 +1,10 @@
+import factory
```

Comment:
I think it's ok to import the constituent parts of factory boy that we use. It's all we're going to do in this file, so it's Ok to be _all about that_.  Should be a little easier on the eye.

Reply:
Done